src/wallet.js

```diff
@@ -0,0 +1,90 @@
+import { existsSync, readFileSync, writeFileSync } from 'fs'; // Import the fs module for file operations
+import { GetMempool } from './Mempool';
+import { Transaction, UnsignedTransaction } from './Transaction';
+
+
+class Wallet {
+    constructor() {
+        if (existsSync('private_key.json')) {
+            const { PrivateKey, Password } = this.loadFromFile();
+            this.PrivateKey = PrivateKey;
+            this.Password = Password;
+        } else {
+            this.Password = this.generatePassword();
+            this.PrivateKey = this.generatePrivatePemString(this.password);
+            this.saveToFile();
+        }
+        this.PublicKey = this.generatePublicPemString(this.PrivateKey, this.Password);

```

Comment:
Why do we need to save variable `PublicKey`? We don't use it anywhere after saving

Reply:
Also better to do error handling to check if private key and password are not empty